app/views/warehouse_reports/bed_utilization/_table.haml

```diff
@@ -24,7 +24,7 @@
         - @report.report.each do |id, project|
           - next unless project
           %tr
-            %td= link_to project.name, project_path(project.id)
+            %td= link_to_if current_user.can_access_project?(project), project.name(current_user), project_path(project.id)
```

Comment:
Turns out `project` isn't a project record here, but an `OpenStruct`.  We'll need to re-work the controller/model to pull projects instead of plucking, or to pick the appropriate name before it gets to the view.